db/migrate/20210824160127_update_notified_at_with_created_at.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateNotifiedAtWithCreatedAt < ActiveRecord::Migration[6.1]
+  def change
+    reversible do |dir|
+      dir.up do
+        PlanningApplication.all.map do |planning_application|
+          planning_application.validation_requests.map do |request|
+            request.update(notified_at: request.created_at.strftime("%e %B %Y"))
```

Comment:
We don't need to format the date inside the table - only when we display it

Reply:
I had checked the date if the notified_at within my db and realised they were all formatted already so thought it was expected hence placing it here.. but I'll change it!